### PR TITLE
Various patches

### LINKED ISSUE #69: Joe and Andrew - WIP tasks
- [ ] add kubectl patch to metro
- [x] move storkctl install to px-install (and any other scripts that install px, eg cp4d)
- [x] move aws stuff out of async-dr
- [x] move aws stuff out of metro
- [x] separate licence script
- [x] dont apply async dr to remote clusters

### PR DESCRIPTION
**Pull request by grdnrio**
Closes #69 _giggidy_

### CODE REVIEW

File: assets/app-migration.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: default
 spec:
   # This should be the name of the cluster pair created above
-  clusterPair: remotecluster
+  clusterPair: remotecluster-2
```
Comment (andrewh1978) [inline assets/app-migration.yml:8]: Do we really want to mention the cluster number here? If we don't, it can be used on any cluster.

Comment (grdnrio) [reply assets/app-migration.yml:8]: I'm aligning it with the asset here - https://github.com/andrewh1978/px-deploy/blob/joeg/patches/assets/app-migration.yml

Comment (grdnrio) [reply assets/app-migration.yml:8]: We don't, but the only migration demo we have is from cluster 1 to cluster 2, meaning every time I run that demo I ahve to remember to edit this template before I apply it.

File: scripts/licenses [hunk 2]
```diff
@@ -0,0 +1,5 @@
+for i in $licenses; do
```
Comment (andrewh1978) [inline scripts/licenses:1]: Could do with validating the variable and licence IDs

**Pull request merged**